include/swift/SIL/SILNode.h

```diff
@@ -114,7 +115,7 @@ class SILNodePointer {
 ///   subobject.  If the SILNode is actually the base subobject of a
 ///   ValueBase subobject, the cast will yield a corrupted value.
 ///   Always use the LLVM casts (cast<>, dyn_cast<>, etc.) instead.
-class alignas(8) SILNode {
+class alignas(8) SILNode : public SwiftObjectHeader {
```

Comment:
Can you add a comment here explaining that SILNode inherits from SwiftObjectHeader to allow for libswift bridging/ to look for information in SwiftObjectHeader.h about how this all works?

Reply:
I'll mention libswift/README.md, where this is explained